src/base/IoManip.h

```diff
@@ -76,27 +76,31 @@ operator <<(std::ostream &os, const RawPointerT<Pointer> &pd)
 }
 
 /// std::ostream manipulator to print integers as hex numbers prefixed by 0x
+/// Normally used through the asHex() convenience function
 template <class Integer>
 class AsHex
```

Comment:
Please consider applying the new functionality to std::hex code inside Eui::Eui48::lookup(), Rock::SwapDir::createStoreIO(), Rock::SwapDir::createUpdateIO(), Rock::SwapDir::openStoreIO() and other contexts. Search for "std::hex" and "std::setfill". Keep std::uppercase.

I do not insist on this application, only consideration.

Reply:
For Eui things, I think the best solution is to implement an operator<< for the relevant types, which is
out of scope here.

I've gone over the list of std::hex users, I think I caught all those for which it makes sense right away.